### ISSUE TITLE
Does not work from Firefox

### ISSUE THREAD

**Issue by notramo** [element-hq/element-call#964]
### Steps to reproduce

Use call.element.io from Firefox.

#### What did you expect?
Call works.
Maybe analytics record the call duration, and after the call, I can give feedback in a textbox.

#### What happened instead?
Call says it's connected, but there are no audio.
After the nonexistent call, I can not give feedback, because there are no text box.
Analytics doesn't record that the call failed, so you think that everything's OK. It also doesn't record that we went back to Google Meet.

### Operating system

Void Linux

### Browser information

Firefox (any version)

### URL for webapp

call.element.io

### Will you send logs?

No

**Comment by Half-Shot** [comment]
This is quite old and we have no logs, empirically things work on FF these days.

**Comment by notramo** [comment]
Yes, I have been using it successfully on Firefox for a while, I just forgot to close this issue.
